library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -47,4 +49,16 @@ object Predef:
    */
   extension [T](x: T | Null) inline def nn: x.type & T =
     scala.runtime.Scala3RunTime.nn(x)
+
+  /** Enables an expression of type `T|Null`, where `T` is a subtype of `AnyRef`, to be checked for `null`
+   *  using `eq` and `ne`, rather than only `==` and `!=`. This is needed because `Null` no longer has
+   *  `eq` or `ne` methods, only `==` and `!=` inherited from `Any`. */
```

Comment:
I think comments need to be on individual methods, not sure if scaladoc/IDEs pick up anything from the extension block.